### PR TITLE
chore: update component and package generators

### PR DESCRIPTION
**Pull request by jimmynotjim**
I noticed the generators were a little out of date so I updated them over the weekend. Should help maintain some consistency going forward.

### CODE REVIEW

File: apps/pixeloven/site/docs/getting-started/generator.mdx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+```
+yarn pixeloven generate
+```
+
+The generators are somewhat opinionated and make a few assumptions about the organization of your poject. See the instructions for each individual generator below for specifics.
+
+## Applications
+
+1. To start a new app, run the `generate` command from the root of your project and choose `App` from the selection menu.
+1.
```
Comment (m-martinez) [inline apps/pixeloven/site/docs/getting-started/generator.mdx]: Rogue list item here.

File: apps/pixeloven/site/docs/getting-started/generator.mdx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## Components
+
+1. To create a new component, run the `generate` command from the root of the host app or package and choose `Component` from the selection menu.
+1. Choose the appropriate atomic type for your component. Refer to the component docs to learn more about atomic design and how it's implemented in Pixeloven.
+1. Choose the format of the component, we recommend Functional components if you're using React 16.4.x or later.
+1. Choose to manage state within your component. If you're unsure, choose `N`. It's easy enough to add this later if necessary.
+1. Choose to include styles with your component. If you're unsure, choose `N`. It's easy enough to add this later if necessary.
+1. Enter a component name following the naming conventions in the contributing docs.
+1. Enter a short descrption of what your component does. You can expand on the description in the component README.
```
Comment (m-martinez) [inline apps/pixeloven/site/docs/getting-started/generator.mdx]: Typo `descrption` -> `description`

File: apps/pixeloven/site/docs/getting-started/generator.mdx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+```
+
+The generators are somewhat opinionated and make a few assumptions about the organization of your poject. See the instructions for each individual generator below for specifics.
+
+## Applications
+
+1. To start a new app, run the `generate` command from the root of your project and choose `App` from the selection menu.
+1.
+
+## Packages
```
Comment (m-martinez) [inline apps/pixeloven/site/docs/getting-started/generator.mdx]: I suggest re-ordering these sections based on how common it is for a user to have to create a package vs a new component.

File: apps/pixeloven/site/docs/getting-started/generator.mdx (hunk 1 above)
Comment (jimmynotjim) [reply apps/pixeloven/site/docs/getting-started/generator.mdx]: Oh woops, I'm still writing this generator so I don't have anything yet. I'll clear it for now and add apps when that PR lands.

File: apps/pixeloven/site/docs/getting-started/generator.mdx (hunk 3 above)
Comment (jimmynotjim) [reply apps/pixeloven/site/docs/getting-started/generator.mdx]: oh good point, we should reorder the generator as well

File: apps/pixeloven/site/docs/getting-started/generator.mdx (hunk 2 above)
Comment (jimmynotjim) [reply apps/pixeloven/site/docs/getting-started/generator.mdx]: 🤦‍♂

File: apps/pixeloven/site/docs/getting-started/generator.mdx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+## Overview
+
+To start any of the generators run the `generate` command
+
+```
+yarn pixeloven generate
+```
+
+The generators are somewhat opinionated and make a few assumptions about the organization of your poject. See the instructions for each individual generator below for specifics.
```
Comment (m-martinez) [inline apps/pixeloven/site/docs/getting-started/generator.mdx]: `poject` -> `project`

File: apps/pixeloven/site/docs/getting-started/generator.mdx [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+1. To start a new app, run the `generate` command from the root of your project and choose `App` from the selection menu.
+1.
+
+## Packages
+
+1. To start a new package, run the `generate` command from the root of your project and choose `Package` from the selection menu.
+1. Enter a package name following the naming conventions in the contributing docs.
+1. Enter a namespace for the package if necessary. Namespaces are useful to discern packages of similar names across registries like NPM. For example all Pixeloven packages are prefixed with a `@pixeloven` namespace.
+1. Enter a short descrption of what your package does. This will be used in the package.json manifest so keep it short. You can expand on the description in the package README.
```
Comment (m-martinez) [inline apps/pixeloven/site/docs/getting-started/generator.mdx]: `descrption` -> `description`

File: apps/pixeloven/site/docs/getting-started/generator.mdx [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+1. Enter a license type. This is intentially left open, there are too many license types available, but we suggest MIT for it's comprehensive coverage and standard use across the open source community.
```
Comment (m-martinez) [inline apps/pixeloven/site/docs/getting-started/generator.mdx]: `intentially` -> `intentionally`

Comment (m-martinez) [inline apps/pixeloven/site/docs/getting-started/generator.mdx]: `it's` -> `its`

File: apps/pixeloven/site/docs/getting-started/generator.mdx (hunk 3 above)
Comment (jimmynotjim) [reply apps/pixeloven/site/docs/getting-started/generator.mdx]: done

File: apps/pixeloven/site/docs/getting-started/generator.mdx [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+```
+yarn pixeloven generate
+```
+
+The generators are somewhat opinionated and make a few assumptions about the organization of your project. See the instructions for each individual generator below for specifics.
+
+## Components
+
+1. To create a new component, run the `generate` command from the root of the host app or package and choose `Component` from the selection menu.
```
Comment (ductiletoaster) [inline apps/pixeloven/site/docs/getting-started/generator.mdx]: Nit: on the numbers isn't there another syntax for auto numeration?

File: apps/pixeloven/site/docs/getting-started/generator.mdx (hunk 3 above)
Comment (m-martinez) [reply apps/pixeloven/site/docs/getting-started/generator.mdx]: Looks like this is resolved now

File: apps/pixeloven/site/docs/getting-started/generator.mdx (hunk 7 above)
Comment (jimmynotjim) [reply apps/pixeloven/site/docs/getting-started/generator.mdx]: I don't know of any other than using `1.` for every item in markdown. Maybe you're thinking of Bitbucket's own thing? https://www.markdownguide.org/basic-syntax#lists

**Pull request merged**